Extends open hours

BekaValentine opened the pull request:
This PR adds in the code that @rizend wrote to implement the extension of RFID open hours, making them now 10:00-23:00.

Pursuant to meeting discussions here:

[First discussed](https://www.noisebridge.net/wiki/Meeting_notes_2018_03_06#As_open_as_possible_--_mod_hours_from_11:00-22:00_to_10:00-23:00)

[R mentions they will be deploying](https://www.noisebridge.net/wiki/Meeting_notes_2018_03_20)

Commit:
Update timing

Commit:
Even more time

Commit:
<3 tests :)

Commit:
<3 tests!

Commit:
Fix tests again because git

hzeller commented:
I think my question is: do we know that at all times we have a Philanthropist or Member available after 23:00 ? Because we need to make sure to close the space properly, something a regular user has not instructions to do.
If not, we should probably discuss the more open 'last philanthropist/member closes the door' policy, which can be way after midnight or before that. So roughtly the idea that a philanthropist can go to the terminal and say "I am here for another hour or so" or "ok, now I am leaving". In that period, regular user tokens would just work without extra door opening process.

That way, we can keep the space open as long as possible, but keep things safe. I guess that needs to be discussed.

BekaValentine commented:
Regarding member/philanthropist presence: at the time we previously discussed the change, there was consensus that it looked like we were active enough late at night, and we (iinm) addressed all the concerns people had about the earlier mornings. But in the current moment, we are *well* staffed, so to speak. Late nights are so active at Noisebridge at its rare for there to be fewer than three or four philanthropists in the space until well after midnight. I'm often there till 4 or 5 am and about half the time, other people are still there till 2am, 3am. Last night when we left at 2:30am or 3:00am, there were I think 5 philanthropists still there, and maybe only one or two daytime users. Noisebridge is in a pretty good state. The last 7 or so months of being open from 10:00 to 23:00 I think demonstrate that we function perfectly find with that amount of openness.
